software/base/src/main/java/brooklyn/entity/basic/SoftwareProcessImpl.java

```diff
@@ -442,11 +442,12 @@ protected final void callStartHooks() {}
      * plus any ports defined with a config keys ending in .port 
      */
     protected Collection<Integer> getRequiredOpenPorts() {
-        Set<Integer> ports = MutableSet.of(22);
+        // TODO: Should only open 22 *or* 5985. Perhaps a flag / ConfigKey on SoftwareProcessImpl?
+        Set<Integer> ports = MutableSet.of(22, 5985, 3389);
```

Comment:
Hm, tricky to figure out whether it's going to be Windows or Linux from the location. I think the entity impl needs to tell us (e.g. with a method that is overridden by the windows sub-class(es)).

Can you at least add to the TODO what 5985 and 3389 are for.